python/spglib/spglib.py

```diff
@@ -87,6 +89,10 @@
         tuple[Lattice, Positions, Numbers] | tuple[Lattice, Positions, Numbers, Magmoms]
     )
 
+warnings.filterwarnings(
+    "once", category=DeprecationWarning, message=r"dict interface.*"
+)
```

Comment:
Oh, I think this one should be:
```suggestion
warnings.filterwarnings(
    "module", category=DeprecationWarning, message=r"dict interface.*"
)
```

> "module" - print the first occurrence of matching warnings for each module where the warning is issued (regardless of line number)

https://docs.python.org/3/library/warnings.html#the-warnings-filter

The part I'm not sure here, is does the user have a way to silence the warnings? Proper way would be to silence them by not using the dict interface :D